.github/stale.yml

```diff
@@ -0,0 +1,26 @@
+# ProBot Stale Bot
+# https://probot.github.io/apps/stale/
+
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
```

Comment:
I think we could extend this number a little more, maybe two months?

Reply:
Hrm, my idea with this bot is not "just close the issues" but also "alert us that we are not paying attention to them". If we find that 45 is to early and distracting, we can increase this number later.

Anyway, if they are correctly labeled (Accepted, Desing decision, etc) it won't affect/distract us at all, though.